export const newMessage = async(req,res)=>{
    try{

    }
    catch(err){
        
    }
}